### PR TITLE
Add ability to customize and translate resource name

### PR DESCRIPTION
**Pull request by bricesanchez**

### CODE REVIEW

File: resources/config/locales/en.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ en:
           or: or
           replace: " replace it with this one..."
           maximum_file_size: The maximum file size is %{bytes}.
+          resource_title: Title
+          resource_title_help: Extra information about the file
```
Comment (anitagraham) [inline resources/config/locales/en.yml]: Can I suggest that `resource_title_help` be "The title visible to users"?

Maybe?

**Comment by anitagraham** [comment]
I guess resources don't require an alt attribute.

File: resources/config/locales/en.yml (hunk 1 above)
Comment (bricesanchez) [inline resources/config/locales/en.yml]: Yes it will be better !

**Comment by bricesanchez** [comment]
No i don't think resources require an alt attribute. :)

**Comment by bricesanchez** [comment]
output_ebnhpj

**Comment by parndt** [comment]
I don't see any tests specific to this feature?

**Comment by bricesanchez** [comment]
Is someone could help me to test this PR ?

**Comment by anitagraham** [comment]
Will have a look @bricesanchez .  Refinerycms-pages has testing for translated page titles. That's where I am going to look.

**Comment by anitagraham** [comment]
In some places you use `resource_title`, in others `title`.

**Comment by bricesanchez** [comment]
Yes, i've tried to follow how it worked before. Do you have any advice in order to simplify the feature ?

**Comment by anitagraham** [comment]
@bricesanchez Here is a test for the helper. It mostly comes from refinerycms-pages tests for translated page titles. 
````ruby
# Encoding: UTF-8
#/refinerycms/resources/spec/helpers/refinery/admin/resources_helper_spec.rb

require "spec_helper"

module Refinery
  module Admin
    describe ResourcesHelper do
      let(:resource) { FactoryGirl.build(:resource) }

      before do
        Globalize.with_locale(:en) do
          resource.resource_title = "draft"
          resource.save!
        end

        Globalize.with_locale(:lv) do
          resource.resource_title = "melnraksts"
          resource.save!
        end
      end

      describe "#resource_title_with_translations" do
        context 'when the resource has a title in the current locale' do
          it "returns the resource title" do
            expect(helper.resource_title_with_translations(resource)).to eq("draft")
          end
        end

        context 'when the resource does not have a title in the current locale' do
          it 'returns the title from the next locale with a title' do
            Resource.translation_class.where(:locale => :en).first.destroy
            expect(helper.resource_title_with_translations(resource)).to eq("melnraksts")
          end
        end
      end
    end
  end
end
````
This line `Resource.translation_class.where(:locale => :en).first.destroy` doesn't seem to work.
Today I don't have time to follow it up any further. 

I wonder if it would be better to just use the default title (file name) in locales without a translation.

**Comment by bricesanchez** [comment]
@parndt Updated with tests :D
@anitagraham You're right! Now, i display the filename if there is no translation.

**Pull request merged**